lib/matplotlib/tests/test_cm_stubs.py

```diff
@@ -0,0 +1,36 @@
+import matplotlib.pyplot as plt
+import re
+from pathlib import Path
+from matplotlib.colors import Colormap
+
+
+def test_cm_stub_matches_runtime_colormaps():
+    runtime_cm = plt.cm
+    runtime_cmaps = {
+        name
+        for name, value in vars(runtime_cm).items()
+        if isinstance(value, Colormap)
+    }
+
+    cm_pyi_path = Path(__file__).parent.parent / "cm.pyi"
+    assert cm_pyi_path.exists(), f"{cm_pyi_path} does not exist"
+
+    with cm_pyi_path.open("r", encoding="utf-8") as f:
+        pyi_content = f.read()
+
+    stubbed_cmaps = set(
+        re.findall(r"^(\w+):\s+colors\.Colormap", pyi_content, re.MULTILINE)
+    )
+
+    assert runtime_cmaps, (
+        "No colormaps found at runtime — is matplotlib installed correctly?"
+    )
+    assert stubbed_cmaps, (
+        "No colormaps found in cm.pyi — is the file populated?"
```

Comment:
```suggestion
        "No colormaps found in cm.pyi"
```
I‘d leave out the question here as well. It‘s natural to look into the file if we claim something is not found in there.